bilge-impl/src/bitsize_internal/struct_gen.rs

```diff
@@ -157,8 +157,8 @@ pub(crate) fn generate_getter_inner(ty: &Type, is_getter: bool) -> TokenStream {
                 cursor = cursor.wrapping_shr(size as u32);
                 // cast the element value (e.g. u32 -> u8),
                 let raw_value: BaseIntOf<#ty> = raw_value as BaseIntOf<#ty>;
-                // which allows it to be used here (e.g. u4::new(u8))
-                let elem_value = <#ty as Bitsized>::ArbitraryInt::new(raw_value);
+                // which allows it to be used here (e.g. u4::masked_new(u8))
+                let elem_value = <#ty as Bitsized>::ArbitraryInt::masked_new(raw_value);
```

Comment:
you could remove the `& mask` above, possibly making the code slightly more efficient

Reply:
that's a nice new API on arbitrary int :)